pubsub/tests/system.py

```diff
@@ -348,7 +348,9 @@ def test_create_snapshot(self):
 
         # There is no GET method for snapshot, so check existence using
         # list
-        retry = RetryResult(lambda result: result, max_tries=4)
+        def retry_predicate(result):
+            return len(result) > len(before_snapshots)
+        retry = RetryResult(retry_predicate, max_tries=5)
```

Comment:
@lukesneeringer Can you add an empty line between your function and the code that follows it?